PYTHON_STYLE_GUIDE.md

```diff
@@ -68,7 +68,7 @@ we use the `format()` version. The [official Python documentation says](https://
 ## Runnng the Flake8 Style Checker
 
 We use [Flake8](http://flake8.pycqa.org/en/latest/index.html) to check our Python code style. Once you have it installed, you can run it using:
-```text
+```bash
```

Comment:
I generally prefer using `text` to `bash` because GitHub's syntax coloring for `bash` is sometimes absurd.